pkgs/tools/filesystems/dosfstools/default.nix

```diff
@@ -11,6 +11,13 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-2gxB0lQixiHOHw8uTetHekaM57fvUd9zOzSxWnvUz/c=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/dosfstools/dosfstools/commit/77ffb87e8272760b3bb2dec8f722103b0effb801.patch";
```

Comment:
Can you add a comment explaining where you got this patch (from a PR, from `master`, etc)?

Reply:
done.